Analysis/config/2016/sources.cfg

```diff
@@ -12,7 +12,7 @@ categories_base: 2j
 sub_categories_base: NoCuts
 #categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 2j1b+_VBF 2j1b+_VBFT 2j1b+_VBFL 2j0b 2j1b 2j2b+ 2j0Lb 2j1Lb 2j2Lb+ 2j0Tb 2j1Tb 2j1Tb+ 2j2Tb+
 #categories: 2j 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 2j1b+_VBF
-categories: 2j 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 2j1b+_VBF 2j1b+_VBF_qqHH 2j1b+_VBF_ggHH 2j1b+_VBF_TT_L 2j1b+_VBF_TT_FH 2j1b+_VBF_ttH 2j1b+_VBF_DY
+categories: 2j 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 2j1b+_VBF 2j1b+_VBF_qqHH 2j1b+_VBF_ggHH 2j1b+_VBF_TT_L 2j1b+_VBF_TT_FH 2j1b+_VBF_TT 2j1b+_VBF_ttH 2j1b+_VBF_DY
```

Comment:
remove 2j1b+_VBF_TT_L and 2j1b+_VBF_TT_FH from the list (here and for the other years)